workspaces/liferay-sample-workspace/client-extensions/liferay-sample-etc-cron/build.gradle

```diff
@@ -17,6 +17,20 @@ apply plugin: "com.liferay.source.formatter"
 apply plugin: "java"
 apply plugin: "org.springframework.boot"
 
+bootRun {
+	String virtualInstanceId = project.hasProperty("virtualInstanceId") ? project.getProperty("virtualInstanceId") : "default"
+	environment "LIFERAY_ROUTES_CLIENT_EXTENSION", "${gradle.liferayWorkspace.homeDir}/routes/${virtualInstanceId}/${project.name}/"
+	environment "LIFERAY_ROUTES_DXP", "${gradle.liferayWorkspace.homeDir}/routes/${virtualInstanceId}/dxp/"
+	new File("${gradle.liferayWorkspace.homeDir}/routes/${virtualInstanceId}/${project.name}/").listfiles().each { file ->
+		println file.name
+		println file.text
+	}
+	new File("${gradle.liferayWorkspace.homeDir}/routes/${virtualInstanceId}/dxp/").listfiles().each { file ->
+		println file.name
+		println file.text
+	}
```

Comment:
lingering debug code?

Reply:
Yeah i have it in there because it wasn't working last time I tried, so I wanted to have this debug.